net-firewall/fwknop/fwknop-2.6.9-r1.ebuild

```diff
@@ -1,4 +1,4 @@
-# Copyright 1999-2016 Gentoo Foundation
```

Comment:
Also, I think it would be reasonable to elaborate in this commit message a bit why it's done.

Reply:
Are we looking at the same commit?

Reply:
It was `We will enable server USE by default`, which IMHO was enough.
Now it is `We will enable server USE by default and fix REQUIRED_USE instead.`
Not sure what else to add here.